src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -99,16 +96,149 @@ La decisione di non inserire un utente con privilegi è da ricollegare al fatto
       + interagisce con la mappa per visualizzare lo stato di una singola cella [UC2];
       + interagisce con la mappa per selezionare un singolo sensore [UC4].
 
-/ UC1.2 : - Visualizzazione di indicatori numerici
+=== UC1.2 - Visualizzazione di indicatori numerici
   - *Attore Principale*: utente generico.
   - *Precondizioni*: almeno uno dei sensori è attivo e trasmette dati.
   - *Postcondizioni*: il sistema mostra i valori correnti di tutti i sensori.
   - *Scenario Principale*:
     + l'utente accede alla Dashboard; 
     + vengono mostrati pannelli con indicatori numerici che rappresentano i dati attuali misurati da ciascun tipo di sensore distribuito nella città;
     + gli indicatori numerici si aggiornano automaticamente per riflettere i dati più recenti provenienti dai sensori.
+  - *Inclusioni*:
```

Comment:
- come mai hai scelto di metterli come "inclusioni" se posso chiedere?
- di fatto non saranno tutti numerici gli indicatori utilizzati per mostrare i dati di vario tipo, ad esempio quelli pertinenti allo stato del cielo molto probabilmente non lo saranno.. cambierei il titolo dell'UC1.2 in qualcosa di piu' generico a questo punto, della serie "Visualizzazione di dati aggregati"

Reply:
Per quanto riguarda ad inclusioni, perchè abbiamo usato le parole "Specializzazioni" e "Estensioni", l'altra categoria è inclusione e quindi "Inclusioni".